administration/src/components/applications/ApplicationsOverview.tsx

```diff
@@ -0,0 +1,86 @@
+import {Button, Card, H4, Icon} from "@blueprintjs/core";
+import {format} from "date-fns";
+import React from "react";
+import {getApplications_applications as Application} from "../../graphql/applications/__generated__/getApplications";
+import {AppToaster} from "../AppToaster";
+import downloadDataUri from "../../util/downloadDataUri";
+
+interface Props {
+    applications: Application[],
+    token: string
+}
+
+interface JsonField {
+    name: string,
+    translations: { de: string },
+    type: 'Array' | 'String' | 'Number' | 'Boolean' | 'Attachment',
+    value: any
+}
+
+function JsonFieldView(props: { jsonField: JsonField, baseUrl: string, token: string }) {
```

Comment:
Hm, so we build the view completely from user data? Probably too late to change this design decision, but maybe we should at least have some kind of warning on the page.
Just imagine an user building sending a malicious application building a UI here that says something like "Sorry, you have to verify again that you are allowed to see this application due to privacy reasons. Head over to myevildomain.com/login and type your credentials there". There are enough users trusting such emails; those will trust texts shown in our web application even more.